packages/nodejs-ext/scripts/report.js

```diff
@@ -11,8 +12,8 @@ function createReport() {
       packagePath: path.join(__dirname, "/../ext/"),
       architecture: process.arch,
       target: process.platform,
-      muslOverride: Boolean(process.env["APPSIGNAL_BUILD_FOR_MUSL"]),
-      library_type: "static"
+      muslOverride: hasMusl(),
```

Comment:
```suggestion
      muslOverride: (process.env["APPSIGNAL_BUILD_FOR_MUSL"] === "true") || hasMusl(),
```

- Saw a bug in my own code here, `Boolean(process.env["APPSIGNAL_BUILD_FOR_MUSL"])` converts `"false"` to `true`, so now this will give the correct result.
- I think it's a good idea that we still have the `"APPSIGNAL_BUILD_FOR_MUSL"` env variable as an override just in case, to keep it consistent with the other integrations. This change will ensure `hasMusl()` is called when `"APPSIGNAL_BUILD_FOR_MUSL"` is not set. Does that seem reasonable to you?